src/Gir/Marshal/Namespace.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Gir
+{
+	public partial class Namespace
+	{
+		public IEnumerable<ISymbol> GetSymbols()
+		{
+			foreach (var en in Enumerations)
+				yield return en;
+
+			foreach (var alias in Aliases)
+				yield return alias;
```

Comment:
Do bitfields need to be here too? Are we going to just duplicate the list of foreach items?

Reply:
Yes, everything that needs to be in the symbol table has to be specified here

Reply:
I have an idea to not have to specify this.